rackcity/utils/asset_utils.py

```diff
@@ -409,8 +408,9 @@ def save_all_connection_data(data, asset, user, change_plan=None):
     return warning_message
 
 
-def save_all_field_data(data, asset, change_plan=None):
+def save_all_field_data(data, asset, change_plan=None, create_asset_cp = False):
```

Comment:
create_asset_cp will be True if you are on a change plan AND are modifying an aset on this change plan for the first time